dspace-server-webapp/src/main/java/org/dspace/app/rest/model/ItemRest.java

```diff
@@ -75,6 +77,7 @@ public class ItemRest extends DSpaceObjectRest {
     private Date lastModified = new Date();
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
     private String entityType = null;
+    private CollectionRest owningCollection;
```

Comment:
you must use the embedding of the owningCollection link if you need to access this information

Reply:
@abollini Your suggestion makes totally sense to me and I'm align with it, but considering it now implies to "throw to trash" most of the work done in the PR. Because this PR implementation lead us in another way, I think that lead us to the discussion if we agree to bring the feature to DS7.6 kind of as is and do a refactoring later, or just leave it to DS8 with a different implementation.

Reply:
Is this really needed? @pilasou @jeffmorin